### PR TITLE
Release/0.4.3

### PR DESCRIPTION
**Pull request by RedVeil**
- adds info for locked pop
 - uses bn-onboard to connect with Injected and WalletConnect

### CODE REVIEW

**Comment by RedVeil** [comment]
Known Issue: WalletConnect with mobile Metamask will sometimes fail for some reason. It works on all other mobile wallets. Additionally it works with MetmaskMobile in its browser.

File: packages/app/components/NavBar/GetPopMenu.tsx [hunk 1]
```diff
@@ -37,41 +40,43 @@ const GetPopMenu: React.FC<GetPopMenuProps> = ({ chainId }) => {
...
-                await window.ethereum.request({
-                  method: "wallet_watchAsset",
-                  params: {
-                    type: "ERC20",
-                    options: {
-                      address: getPopAddress(chainId),
-                      symbol: "POP",
-                      decimals: 18,
-                      image: "https://popcorn.network/images/icons/pop_64x64.png",
+        {metaMaskConnected && (
```
Comment (anthonymartin) [inline packages/app/components/NavBar/GetPopMenu.tsx:53]: @RedVeil do you know why we only show this with metamask and not walletconnect?

File: packages/app/helper/web3Onboard.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      {
+        id: ChainIdHex.Localhost,
+        token: "ETH",
+        label: "Localhost",
+        rpcUrl: RPC_URLS[ChainId.Localhost],
+      },
+    ],
+    appMetadata: {
+      name: "Popcorn",
+      icon: "<svg></svg>",
```
Comment (anthonymartin) [inline packages/app/helper/web3Onboard.tsx:61]: Is there an icon that we can use here?

File: packages/app/tailwind.config.js [hunk 3]
```diff
@@ -1,123 +1,120 @@
 const colors = require("tailwindcss/colors");
 
 module.exports = {
-  content: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}", "./containers/**/*.{js,ts,jsx,tsx}"],
-  theme: {
-    screens: {
-      xs: "440px",
-      // => @media (min-"440px) { ... }
+	content: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}", "./containers/**/*.{js,ts,jsx,tsx}"],
```
Comment (anthonymartin) [inline packages/app/tailwind.config.js:4]: what's up with the spacing here? was the formatter not applied?

**Review by anthonymartin — APPROVED** [review]

**Pull request merged**